docs/contributing.md

```diff
@@ -25,7 +25,9 @@ We welcome contributions in the form of comments, issues, or pull requests with
 
 ## Environment setup
 
-1. Use the node environment manager of your choice, but make sure you have the required version specified in `.node-version`. We recommend using [nodenv](https://github.com/nodenv/nodenv) to manage your node versions, but you can also use [homebrew](https://brew.sh/). More info can be found here: [how to install Node.js](https://nodejs.dev/how-to-install-nodejs)
```

Comment:
The nodejs link was a 404, so I removed it.

Additionally, it is not recommended to use brew for Node, as brew prefers to use `latest` whenever possible. Contributors using brew will likely continue to use it (and hopefully know how to deal with those quirks), so I think it best to recommend a version manager and leave it at that.

Reply:
Something I've wondered for a while about asdf is that it supports `.node-version` file, is it better to have two files or just let asdf users know how to enable `legacy_version_file` for asdf? 

https://github.com/asdf-vm/asdf-nodejs#nvmrc-and-node-version-support

Reply:
@gidjin did y'all ever chat about this?